src/elements/pages/dev/game-namespace-tile.ts

```diff
@@ -0,0 +1,41 @@
+import { LitElement, TemplateResult, html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import styles from './game-banner.scss';
+import * as cloud from '@rivet-gg/cloud';
+import { cssify } from '../../../utils/css';
+import assets from '../../../data/assets';
+import routes from '../../../routes';
+
+@customElement('game-namespace-tile')
+export default class GameNamespaceTile extends LitElement {
+	static styles = cssify(styles);
+
+	@property({ type: String })
+	gameId: string;
+
+	@property({ type: Object })
+	namespace: cloud.NamespaceSummary;
+
+	@property({ type: Object })
+	version: cloud.VersionSummary;
+
+	render() {
+		return html`
+			<a
+				class="m-auto h-full p-2 border-white/10 border flex flex-row place-content-between hover:bg-button-bg-hover-color hover:-translate-y-px transition"
```

Comment:
Does not match corner radius of the button above.
![image](https://github.com/rivet-gg/hub/assets/3322446/7ead8305-15af-4e47-816d-a2014fafce4d)